test/testBraveAlerterConfigurator.js

```diff
@@ -0,0 +1,357 @@
+                expect(this.testStartTimes[this.testLocationId] === this.initialTestStartTime)
+            })
+
+            it('should reset redis for this location', () => {
+                expect(redis.addStateMachineData).to.be.calledWith('Reset', this.testLocationId)
+            })
+        })
+    })
+
+    describe('getReturnMessage', () => {
+        before(() => {
+            const braveAlerterConfigurator = new BraveAlerterConfigurator()
+            const braveAlerter = braveAlerterConfigurator.createBraveAlerter()
+            this.alertStateMachine = braveAlerter.alertStateMachine
+        })
+
+        it('should get message when STARTED => WAITING_FOR_REPLY', () => {
+            const returnMessage = this.alertStateMachine.getReturnMessage(ALERT_STATE.STARTED, ALERT_STATE.WAITING_FOR_REPLY)
+
+            expect(returnMessage).to.equal('Please respond with the number corresponding to the incident. \n1: No One Inside\n2: Person Responded\n3: Overdose\n4: None of the Above')
+        })
+
+        it('should get message when STARTED => WAITING_FOR_CATEGORY', () => {
+            const returnMessage = this.alertStateMachine.getReturnMessage(ALERT_STATE.STARTED, ALERT_STATE.WAITING_FOR_CATEGORY)
+
+            expect(returnMessage).to.equal('Please respond with the number corresponding to the incident. \n1: No One Inside\n2: Person Responded\n3: Overdose\n4: None of the Above')
+        })
+
+        it('should get message when WAITING_FOR_REPLY => WAITING_FOR_CATEGORY', () => {
+            const returnMessage = this.alertStateMachine.getReturnMessage(ALERT_STATE.WAITING_FOR_REPLY, ALERT_STATE.WAITING_FOR_CATEGORY)
+
+            expect(returnMessage).to.equal('Please respond with the number corresponding to the incident. \n1: No One Inside\n2: Person Responded\n3: Overdose\n4: None of the Above')
+        })
+
+        it('should get message when WAITING_FOR_CATEGORY => WAITING_FOR_CATEGORY', () => {
+            const returnMessage = this.alertStateMachine.getReturnMessage(ALERT_STATE.WAITING_FOR_CATEGORY, ALERT_STATE.WAITING_FOR_CATEGORY)
+
+            expect(returnMessage).to.equal('Invalid category, please try again\n\nPlease respond with the number corresponding to the incident. \n1: No One Inside\n2: Person Responded\n3: Overdose\n4: None of the Above')
+        })
+
+        it('should get message when WAITING_FOR_CATEGORY => COMPLETED', () => {
+            const returnMessage = this.alertStateMachine.getReturnMessage(ALERT_STATE.WAITING_FOR_CATEGORY, ALERT_STATE.COMPLETED)
+
+            expect(returnMessage).to.equal('Thank you!')
+        })
+
+        it('should get message when COMPLETED => COMPLETED', () => {
+            const returnMessage = this.alertStateMachine.getReturnMessage(ALERT_STATE.COMPLETED, ALERT_STATE.COMPLETED)
+
+            expect(returnMessage).to.equal('Thank you')
+        })
+
+        it('should get default message if given something funky', () => {
```

Comment:
I like this description 😅

Reply:
Hahaha, gotta make testing fun, right? :wink: